### PR TITLE
confocal: added lasers power interface

### PR DESCRIPTION
**Pull request by agom**
NG-2241

Added properties to confocal items to retrieve power percentage for excitation and sted laser.

Companion to https://github.com/lumicks/bluelake/pull/777

### CODE REVIEW

File: lumicks/pylake/detail/mixin.py [hunk 1]
```diff
@@ -147,3 +147,26 @@ def green_photon_time_tags(self) -> Slice:
     @property
     def blue_photon_time_tags(self) -> Slice:
         return _try_get_or_empty(self._get_photon_time_tags, "Blue")
+
+class ExcitationLaserPower:
+    """Red, green, blue, and sted laser power"""
+    def _get_laser_power(self,name):
```
Comment (JoepVanlier) [inline lumicks/pylake/detail/mixin.py]: nitpick, extra space before name

File: lumicks/pylake/detail/mixin.py [hunk 2]
```diff
@@ -147,3 +147,26 @@ def green_photon_time_tags(self) -> Slice:
...
+        power_data = self.file["Confocal diagnostics"][f"Excitation Laser {name}"]
+        # fetch the timestamp of the last datapoint before the beginning of the item
+        start_time = power_data[:self.start].timestamps[-1]
+        return power_data[start_time:self.stop]
```
Comment (JoepVanlier) [inline lumicks/pylake/detail/mixin.py]: nitpick, extra line break after return

File: lumicks/pylake/detail/mixin.py (hunk 1 above)
Comment (agom) [reply lumicks/pylake/detail/mixin.py]: :+1:

File: lumicks/pylake/detail/mixin.py (hunk 2 above)
Comment (agom) [reply lumicks/pylake/detail/mixin.py]: :+1:

**Pull request merged**